views/js/controller/DeliveryMgmt/wizard.js

```diff
@@ -0,0 +1,280 @@
+         */
+        listTests: function listTests(data) {
+            return new Promise(function(resolve, reject) {
+                $.ajax({
+                    url: urlUtils.route('getAvailableTests', 'DeliveryMgmt', 'taoDeliveryRdf'),
+                    data: {
+                        q: data.q,
+                        page: data.page
+                    },
+                    type: 'GET',
+                    dataType: 'JSON'
+                }).done(function(tests) {
+                    if (tests) {
+                        resolve(tests);
+                    } else {
+                        reject(new Error(__('Unable to load tests')));
+                    }
+                }).fail(function() {
+                    reject(new Error(__('Unable to load tests')));
+                });
+            });
+        }
+    };
+
+    /**
+     * wrapped the old jstree API used to refresh the tree and optionally select a resource
+     * @param {String} [uriResource] - the uri resource node to be selected
+     */
+    var refreshTree = function refreshTree(uriResource){
+        actionManager.trigger('refresh', {
+            uri : uriResource
+        });
+    };
+
+    return {
+        // Builds form elements & defines button actions
+        // TODO: refactor into some logical functions and deduplicate a little
+        start: function() {
+            var $multiForm = $('.multi-form-container');
+            var $switch = $('.form-switch', $multiForm);
+
+            var $compiledForm = $('#simpleWizard');
+            var $ltiForm = $('#simpleLtiWizard');
+
+            var $testFilterContainer = $compiledForm.find('.test-select-container');
+            var $providerFilterContainer = $ltiForm.find('.lti-provider-select-container');
+
+            var $testFormElement = $('#test');
+            var $providerFormElement = $('#ltiProvider');
+
+            var $compiledContainer = $compiledForm.closest('.content-block');
+            var $ltiContainer = $ltiForm.closest('.content-block');
+
+            var $oldCompiledSubmitter = $compiledForm.find('.form-submitter');
+            var $oldLtiSubmitter = $ltiForm.find('.form-submitter');
+
+            var taskCompiledCreationButton;
+            var taskLtiCreationButton;
+
+            $ltiForm.addClass('hidden');
+
+            switchFactory($switch, {
+                on: {
+                    label: __('TAO delivery'),
+                    active: true
+                },
+                off: {
+                    label: __('LTI based delivery'),
+                },
+                monoStyle: true
+            })
+            .on('change', function() {
+                $ltiForm.toggleClass('hidden');
+                $compiledForm.toggleClass('hidden');
+            });
+
+            filterFactory($testFilterContainer, {
+                placeholder: __('Select the test you want to publish to the test-takers'),
+                width: '64%',
```

Comment:
where this width value is coming from?

Reply:
I guess 100% - 35% - 1 ? It's standard on our form layouts.

Reply:
Would be nice to store these in a descriptive variable but for now, it's ok